pages/sdk/data-hooks/useBets.mdx

```diff
@@ -0,0 +1,92 @@
+import { Callout } from 'components'
+
+Used for fetch bets history for specific bettor.
+
+#### Usage
+
+```ts
+import { useBets } from '@azuro-org/data'
+
+const { loading, error, data: bets } = useBets(props)
+```
+
+<Callout type="warning">
+  The hook won't be called until `bettor` value is nullish.
```

Comment:
wrong meaning, should be
"until `bettor` value is defined"
or
"while `bettor` value is nullish"

Reply:
cff9b0a6319353850ff3a3a5808d675a1f955ddc